config/routes.rb

```diff
@@ -1,15 +1,15 @@
 Collections::Application.routes.draw do
+  # Note that this app only receives requests for routes registered with the
+  # content-store (by collections-publisher) - whenever the routes below
+  # change, also change the routes claimed by collections-publisher.
 
   mount JasmineRails::Engine => '/specs' if defined?(JasmineRails)
```

Comment:
While you're tidying this up, it's probably worth wrapping this in an `unless Rails.env.production?` just to make sure we never expose this accidentally.